src/lib/schemas.ts

```diff
@@ -70,8 +70,11 @@ export const detailStoreSchema = portfolioSchema.extend({
 
 export const investmentSchema = z.object({})
 
-export const loginFormSchema = z.object({
-	email: z.string().email({ message: 'Enter a valid email.' }),
+export const emailFormSchema = z.object({
+	email: z.string().email({ message: 'This doesn’t look like a correct email' }),
+})
+
+export const loginFormSchema = emailFormSchema.extend({
 	password: z.string().min(6, { message: 'Password must be at least 6 characters long.' }),
```

Comment:
Should be localised string:
```suggestion
	password: z.string().min(6, { message: $_('Password must be at least 6 characters long.') }),
```